src/commands.js

```diff
@@ -82,9 +83,29 @@ module.exports = {
     bot.on("messageCreate", async msg => {
       if (msg.author.bot) return;
       if (msg.author.id === bot.user.id) return;
-      if (! msg.content) return;
+      if (!msg.content) return;
 
       const matchedCommand = await manager.findMatchingCommand(msg.content, { msg });
+
+      const args = msg.content.slice(config.prefix.length).trim().split(' ');
+      const command = args.shift().toLowerCase();
+      if (command === "shownote") {
+        //BYCOP
+        if (utils.messageIsOnInboxServer(msg)) {
+          if (!args[0]) return (utils.postError(msg.channel, "Usage : " + config.prefix + "shownote + UserID"));
+          let notes = JSON.parse(fs.readFileSync("./logs/notes/notes.json", "utf8"));
```

Comment:
You also seem to be writing all notes to json files, which is practical for plugins but seeing as this is a built-in feature it might be better to use knex and store the notes in the database.

Reply:
Can we talk on discord to solve this ?